src/applications/disability-benefits/all-claims/content/contactInformation.jsx

```diff
@@ -12,7 +12,8 @@ const PhoneViewField = ({ formData: phoneNumber = '', name }) => {
   const phoneString = `${firstPhoneString}-${middlePhoneString}-${lastPhoneString}`;
   return (
     <p>
-      <strong>{name}</strong>: {phoneString}
+      <strong>{name}</strong>:{' '}
+      {phoneNumber.trim().length > 0 ? phoneString : ''}
```

Comment:
I'd recommend using the `<va-telephone>` web component to render the phone number